include/vgamem.h

```diff
@@ -24,6 +24,12 @@
 #ifndef SCHISM_VGAMEM_H_
 #define SCHISM_VGAMEM_H_
 
+#define VGA_CHAR_WIDTH          80
```

Comment:
these names are kind of misleading; I'd suggest `VGAMEM_TEXT_WIDTH` and `VGAMEM_TEXT_HEIGHT`, but I'm not sure if those are easier to understand without proper context. I'm open to other names as well!

Reply:
That's fair :-) What about `VGAMEM_COLUMNS` and `VGAMEM_ROWS`?

Reply:
That sounds good to me